edk2toolext/invocables/edk2_update.py

```diff
@@ -61,6 +62,10 @@ def RetrieveCommandLineOptions(self, args):
         '''  Retrieve command line options from the argparser '''
         super().RetrieveCommandLineOptions(args)
 
+    def GetSkippedDirectories(self):
```

Comment:
why is this needed if the super has the function?

Reply:
Added when debugging something and I forgot to remove it. I'll update it.